### PR TITLE
PR2 - HTML Files

### PR DESCRIPTION
**Pull request by AntimatterReactor**

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -1,30 +1,32 @@
...
 			<p id="tag"><b>Night</b> lets you see the <b>Stars</b></p>
 
 			<div id="actions">
-				<div class="action"><a href="install.html"><h2>Install</h2></a></div>
-				<div class="action"><a href="learn.html"><h2>Learn</h2></a></div>
-				<div class="action"><h2>Extras</h2></div>
+				<div class="action"><a href="index.html"><p class="desc current-page"><b>Home</p></b></a></div>
+				<div class="action"><a href="install.html"><p class="desc"><b>Install</p></b></a></div>
+				<div class="action"><a href="learn.html"><p class="desc"><b>Learn</p></b></a></div>
+				<div class="action"><a href="extras.html"><p class="desc"><b>Extras</p></b></a></div>
```
Comment (DynamicSquid) [inline docs/index.html:29]: what is the `desc` tag for? I think the css for the `action` class has everything already

File: docs/install.html [hunk 2]
```diff
@@ -1,30 +1,31 @@
...
 			<p id="tag">Latest Release <b>1.0.0</b></p>
 
 			<div id="actions">
-				<div class="action"><a href="index.html"><h2>Home</h2></a></div>
-				<div class="action"><a href="learn.html"><h2>Learn</h2></a></div>
-				<div class="action"><h2>Extras</h2></div>
+				<div class="action"><a href="index.html"><p class="desc"><b>Home</p></b></a></div>
+				<div class="action"><a href="install.html"><p class="desc current-page"><b>Install</p></b></a></div>
+				<div class="action"><a href="learn.html"><p class="desc"><b>Learn</p></b></a></div>
+				<div class="action"><a href="extras.html"><p class="desc"><b>Extras</p></b></a></div>
```
Comment (DynamicSquid) [inline docs/install.html:28]: same here, what is the `desc` class used for?

File: docs/install.html [hunk 3]
```diff
@@ -39,9 +40,9 @@ <h1>Install</h1>
 				<li><a href="#linux">Linux</a></li>
 			</ul>
 
-			<h2 id="source">Source</h2>
-			
-			<p class="desc">The source code for Night can be found on the <a href="github.com/DynamicSquid/night">GitHub page</a>.</p>
+			<h2 id="source" class="heading">Source</h2>
```
Comment (DynamicSquid) [inline docs/install.html:43]: I think `h2` is the only `h2` in this HTML page, so we don't need a class for it

File: docs/learn.html [hunk 4]
```diff
@@ -1,43 +1,43 @@
...
 			<p id="tag"><b>Navigating</b> through the <b>Night</b></p>
 
 			<div id="actions">
-				<div class="action"><a href="install.html"><h2>Install</h2></a></div>
-				<div class="action"><a href="index.html"><h2>Home</h2></a></div>
-				<div class="action"><h2>Extras</h2></div>
+				<div class="action"><a href="index.html"><p class="desc"><b>Home</p></b></a></div>
+				<div class="action"><a href="install.html"><p class="desc"><b>Install</p></b></a></div>
+				<div class="action"><a href="learn.html"><p class="desc current-page"><b>Learn</p></b></a></div>
+				<div class="action"><a href="extras.html"><p class="desc"><b>Extras</p></b></a></div>
```
Comment (DynamicSquid) [inline docs/learn.html:28]: same for here as well, what is the `desc` class for?

**Review by DynamicSquid — COMMENTED** [review]
awesome, thanks! I really appreciate this :)

I just left a few comments on some of the files

File: docs/install.html (hunk 3 above)
Comment (AntimatterReactor) [reply docs/install.html:43]: yeah, but unfortunately if you remove that class the h2 becomes black in color and has the default font

**Comment by AntimatterReactor** [comment]
yeah the desc class thing is just a mistake of copy n' paste

File: docs/install.html (hunk 3 above)
Comment (DynamicSquid) [reply docs/install.html:43]: gotcha, thanks for the bug! I fixed it by making `h2` global

**Comment by DynamicSquid** [comment]
I fixed the black header text bug in the master branch (my local one, I'll push it here soon).

I'll also delete the other files, and only keep `extras.html` and `reference.html`, then I'll merge!

**Comment by DynamicSquid** [comment]
Oh whoops, I deleted the file. I only meant to remove it from this PR. Do you know how to do that?

**Pull request merged**